conrpg.cpp

```diff
@@ -127,45 +168,224 @@ int ambush(){
 	Sleep(3000);
 }
 
-int main(){
-	srand(time(0));
+// Space saving
+
+void d1n(){
+	if(a==0) truenarrator("\t  Trees, interesting choice. You come closer to the trees but you don't find anything.", 50);
+	if(a==1) truenarrator("\t  For real? You visit the trees again, and... you don't find anything.", 50);
+	if(a==2) truenarrator("\t  There is something weird about those trees. They're... trees.", 50);
+	if(a>2){
+		truenarrator("\t  You visit the trees once more. Once again, you find nothin-", 50); cout << endl;
+		truenarrator("\t  w- what's that sound? It seems like... you have awakened something. It's angry.", 75); cout << endl; skip(300);
+		truenarrator("\t  Behold Groot, the tree [BOSS] [10M HP] You stand no chance against this beast as it one hits you", 50); cout << endl;

```

Comment:
What about "Mozz"?

Reply:
Okay, Mozz sounds good for now, it can be a placeholder and perhaps the name.

Reply:
Sounds alright. Let's roll with that then.